dev/bots/analyze.dart

```diff
@@ -518,7 +518,7 @@ Future<void> _verifyGeneratedPluginRegistrants(String flutterRoot) async {
     }
   }
 
-  final Set<String> outOfDate = Set<String>();
+  final Set<String> outOfDate = <String>{};
```

Comment:
can't second type specification be omitted?
like: `final Set<String> outOfDate = {};`

Reply:
yes but the internal Flutter style guide requires to always specify types.